mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go.erb

```diff
@@ -1322,6 +1346,26 @@ resource "google_sql_database_instance" "instance" {
 }
 `
 
+var testGoogleSqlDatabaseInstance_insights = `
+resource "google_sql_database_instance" "instance" {
+  name   = "tf-lw-%d"
```

Comment:
```suggestion
  name   = "tf-test-%d"
```
Allows any hanging database instances to be swept (some other test configs in this file are outdated)

Reply:
Fixed https://github.com/GoogleCloudPlatform/magic-modules/pull/4464/commits/42df0e23f30b878825c04b7849efd259e47aaccd